pkg/web/apps/apps_http_handler.go

```diff
@@ -32,3 +33,26 @@ func (a *HTTPHandler) GetApps(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, apps)
 }
+
+// GetAppById returns apps by id as JSON from the AppService
+func (a *HTTPHandler) GetAppById(c echo.Context) error {
+
+	id := c.Param("id")
+	if !IsValidUUID(id) {
+		return httperrors.BadRequest(c, "Invalid id supplied")
+	}
+
+	apps, errMessage := a.Service.GetAppById(id)
+
+	if errMessage == "App Not Found" {
+		return httperrors.NotFound(c, "App Not Found")
+	}
+	return c.JSON(http.StatusOK, apps)
+
+}
+
+// IsValidUUID helper function takes in a string and confirms is a UUID and returns bool
+func IsValidUUID(id string) bool {
```

Comment:
Will this function be used anywhere outside of the handler?

If **yes** then I think it should be moved into a separate package.

If **no** then I think it should be a private function

Reply:
Don't see it being used outside, I guess it should be private